<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameWindow</class>
 <widget class="QMainWindow" name="gameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1005</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1191</width>
      <height>981</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="u.qrc">:/33/gamewindow-2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="clock">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>60</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum1">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>150</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum.qrc">:/new/prefix1/drum- 7.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="drum2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>120</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum_2.qrc">:/drum_2/drum2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="drum3">
    <property name="geometry">
     <rect>
      <x>1054</x>
      <y>158</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum.qrc">:/new/prefix1/drum- 7.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="drum4">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>180</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum_2.qrc">:/drum_2/drum2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="drum5">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>170</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum.qrc">:/new/prefix1/drum- 7.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="drum6">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>170</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum_2.qrc">:/drum_2/drum2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="scorenum">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="scorewin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1241</width>
      <height>951</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="re.qrc">:/all/scorewindow.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="scorescore">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>64</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
background-image: url(:/all/return - 7.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>850</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
background-image: url(:/all/return - 7.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>854</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/all/exit - 3.png);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="drum.qrc"/>
  <include location="re.qrc"/>
  <include location="drum_2.qrc"/>
  <include location="u.qrc"/>
 </resources>
 <connections/>
</ui>
